repro/ResponseContext.cxx

```diff
@@ -584,7 +584,8 @@ ResponseContext::beginClientTransaction(repro::Target* target)
       mRequestContext.getOriginalRequest().method()==INVITE)
    {
       mRequestContext.mInitialTimerCSet=true;
-      mRequestContext.updateTimerC(target->mSipMessageOptions.mTxOptions.mTimerC);
+      int delay = target->mSipMessageOptions ? target->mSipMessageOptions->mTxOptions.mTimerC : 0;
```

Comment:
rename to delay to customDelayMs so that a value of 0 makes more sense when looking at this code :)